src/main/deltachat.js

```diff
@@ -45,26 +45,19 @@ class DeltaChatController {
    */
   constructor (cwd) {
     this.cwd = cwd
-    this.ready = false
-    this.credentials = {
-      email: null,
-      cwd: null
-    }
-    this._selectedChatId = null
-    this._showArchivedChats = false
+    this._resetState()
```

Comment:
Moved this code to `_resetState()` since we need to do this in `logout()` as well. Also `this.credentials.cwd` was not used.